src/utils.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable import/prefer-default-export */
+const todos = JSON.parse(localStorage.getItem('todos'))
+  ? JSON.parse(localStorage.getItem('todos'))
+  : [];
+export const addTodo = (description) => {
+  const index = todos && todos.length > 0 ? todos.length + 1 : 1;
+  const todo = { description, index, completed: false };
+  todos.push(todo);
+  localStorage.setItem('todos', JSON.stringify(todos));
+  window.location.reload();
+};
+
+export const deleteTodo = (index) => {
+  const remainingTodos = todos.filter((todo) => todo.index !== index);
+  if (remainingTodos.length > 0) {
+    remainingTodos.forEach((todo, index) => {
+      todo.index = index + 1;
+    });
```

Comment:
- [OPTIONAL] Let's keep our code DRY. This logic with the same lines of code has been re-written a couple of times in different methods in this module. It's high time it was refactored to its own method. Don't you think so too? 😃.